### PR TITLE
Disable Cache

### PR DESCRIPTION
**Pull request by CalebMorris**
#36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -290,6 +291,13 @@ When rendering multiple `<Autosuggest />`s, make sure to give them unique `id`s.
 <Autosuggest id="destination" suggestions={getDestinationSuggestions} />
 ```
 
+<a name="disableCacheOption"></a>
+#### scrollBar (optional)
```
Comment (moroshko) [inline README.md]: Copy-pasta error?

File: src/Autosuggest.js [hunk 2]
```diff
@@ -16,7 +16,8 @@ export default class Autosuggest extends Component { // eslint-disable-line no-s
     onSuggestionUnfocused: PropTypes.func,  // This function is called when suggestion is unfocused via mouse hover or Up/Down keys
     inputAttributes: PropTypes.object,      // Attributes to pass to the input field (e.g. { id: 'my-input', className: 'sweet autosuggest' })
     id: PropTypes.string,                   // Used in aria-* attributes. If multiple Autosuggest's are rendered on a page, they must have unique ids.
-    scrollBar: PropTypes.bool               // Should be set to true when the suggestions container can have a scroll bar
+    scrollBar: PropTypes.bool,              // Should be set to true when the suggestions container can have a scroll bar
+    disableCache: PropTypes.bool
```
Comment (moroshko) [inline src/Autosuggest.js]: A short description would be nice here

**Comment by moroshko** [comment]
@CalebMorris Thanks a lot for the Pull Request!

I'm curious to know whether you have a use case for not having cache?

Could you also please remove all the `dist` files from the PR? I'll generate them before publishing.
Looks like the PR can't be merged automatically. If removing the `dist` files doesn't fix this, please merge from `master`.

Thanks again for your help!

**Comment by CalebMorris** [comment]
@moroshko I currently use the Reflux/React architecture at work and started adding store level caching.
Since I want to control the logic and state of a site interaction I want this to be inside of the store as part of the pattern.
For a more applied use case I want to control the expiration and cache strategy on a data and component specific case. Having a mixin for the store is easier than having to change for each component.

**Comment by moroshko** [comment]
@CalebMorris That makes sense. 

Are you saying that you are using Autosuggest in a flux architecture? [Others](https://github.com/moroshko/react-autosuggest/issues/27), including myself, would be interested to see how you do this.

**Comment by CalebMorris** [comment]
Added how I use Reflux to ticket

**Comment by CalebMorris** [comment]
Closing for clean merge in #43

**Pull request closed without merging**